src/App.tsx

```diff
@@ -1,11 +1,87 @@
 import './installSesLockdown';
-import 'ses';
+import { motion } from 'framer-motion';
+import { useEffect } from 'react';
+import { useAtom } from 'jotai';
```

Comment:
not familiar with Jotai. worried about YA state library but it seems pretty light

Reply:
Yea, we were using `autodux` in the other dapps which was a lite version of `redux`. This is like a lite version of `recoil` basically. I think the patterns are very react-like and avoid some of the ugliness of relying too much on the context API. But yea, with your blessing I'd like to see how this goes.

Reply:
🤴 ⚔️    gofurit